src/bot.py

```diff
@@ -198,6 +202,22 @@ def on_maintenance(self):
                 wait(0.1, 0.2)
                 self._ui_manager.close_vendor_screen()
                 self._tps_left = 20
+            elif not self.out_of_gold: #You are out of gold. Adding gold to pickit until you can repair
+                wait(0.1, 0.2)
+                self._ui_manager.close_vendor_screen()
+                config_overrides = {"misc_gold":"1","stash_gold":"1"}
+                self._config = Config(False,config_overrides)
+                self._item_finder = ItemFinder(self._config)
+                self._ui_manager = UiManager(self._screen, self._template_finder,self._config)
+                self._pickit = PickIt(self._screen, self._item_finder, self._ui_manager, self._belt_manager, self._game_stats)
+                self.out_of_gold = True
```

Comment:
I dont see where we would ever set this back to False

Reply:
The comment was not changed. The warning says that it will change upon bot restart after destroying and creating a new bot instance.